### PR TITLE
DOC: autoreformat of more docstrings.

### PR DESCRIPTION
**Pull request by Carreau**
Unlike previous PR this is

1) more conservative on whitespace at start and end of docstrings,
2) does visit the full ast, and hence also fix methods.

There are a couple of manual changes where vélin had issues, and used a
custom fork of numpydoc to handle the typoes instead of failing.

### CODE REVIEW

File: napari/_qt/layers/qt_labels_layer.py
Comment (Carreau) [inline napari/_qt/layers/qt_labels_layer.py:312]: This was done manully.

File: napari/resources/__init__.py
Comment (Carreau) [inline napari/resources/__init__.py:68]: The heuristic is : if one section have one blankline somewhere we'll separate items with blanklines.

File: napari/utils/colormaps/vendored/cm.py
Comment (Carreau) [inline napari/utils/colormaps/vendored/cm.py]: this is likely due to the fact that this docstring has no sections with headers otherwise it would leave a blankline.

File: napari/utils/colormaps/vendored/cm.py
Comment (sofroniewn) [inline napari/utils/colormaps/vendored/cm.py]: Just flagging that this change happens inside our `vendored` folder which we might want to ignore. Thoughts on this @jni? We could easily revert the few changes inside vendored files before merging, but your call

**Review by sofroniewn — APPROVED** [review]
LGTM! All changes are great - only question for @jni is if we want to revert changes in the ~2 files inside one of our `vendored` folders

File: napari/utils/colormaps/vendored/cm.py
Comment (Carreau) [reply napari/utils/colormaps/vendored/cm.py]: oh, yeah, I can revert that or redo just the part not in vendor.

Comment (Carreau) [reply napari/utils/colormaps/vendored/cm.py]: done.

File: napari/utils/event.py
Comment (jni) [inline napari/utils/event.py:590]: @Carreau I'm going to merge the PR but this looks like it missed this indentation, maybe because it was "dependent" on another fix?

Comment (jni) [reply napari/utils/event.py:590]: Or maybe this is the "more conservative with whitespace" version...? Incidentally, I've very much liked all of your whitespace fixes so far...

**Review by jni — APPROVED** [review]
Awesome, as before! I love this tool! ❤️

**Pull request merged**
